src/shared/loadPseudoData.c

```diff
@@ -0,0 +1,29 @@
+#include "loadPseudoData.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+int loadData(const char *input_file, const char *output_file) {
+    FILE *ifp, *ofp;
+
+    if ((ifp = fopen(input_file, "r")) == NULL) {
+        printf("Error : Unable to open %s for reading\n", input_file);
+        exit(EXIT_FAILURE);
+    }
+    if ((ofp = fopen(output_file, "w+")) == NULL) {
+        printf("Error : Unable to open %s for reading\n", output_file);
+        exit(EXIT_FAILURE);
+    }
+
+    copyData(ifp, ofp);
+
+    fclose(ifp);
+    fclose(ofp);
+}
+
+void copyData(FILE *ifp, FILE *ofp) {
```

Comment:
We use snake case for function names and lower camel case for parameters #3
`copyData` -> `copy_data`

Also better with longer and more descriptive names.
Maybe consider if this should actually be a public function. We might just want this as a [static function](https://www.tutorialspoint.com/static-functions-in-c#:~:text=A%20static%20function%20in%20C,keyword%20before%20the%20function%20name.) within this .c file. For static functions it is also good to start the name with an `_` so the name becomes `_copy_data`